examples/advanced-pytorch/README.md

```diff
@@ -1,77 +1,46 @@
-```
+This examples shows how to achieve the above using both a `Strategy` as well as the low-level API.
 
-This will create a new directory called `advanced-pytorch` containing the following files:
+This examples uses [Flower Datasets](https://flower.ai/docs/datasets/) with the [Dirichlet Partitioner](https://flower.ai/docs/datasets/ref-api/flwr_datasets.partitioner.DirichletPartitioner.html#flwr_datasets.partitioner.DirichletPartitioner) to partition the [Fashion-MNIST](https://huggingface.co/datasets/zalando-datasets/fashion_mnist) dataset in a non-IID fashion into 50 partitions.
 
-```shell
--- pyproject.toml
--- requirements.txt
--- client.py
--- server.py
--- README.md
--- run.sh
-```
+![](_static/fmnist_50_lda.png)
 
-### Installing Dependencies
+> \[!TIP\]
+> You can use Flower Datasets [built-in visualization tools](https://flower.ai/docs/datasets/tutorial-visualize-label-distribution.html) to easily generate plots like the one above.
 
-Project dependencies (such as `torch` and `flwr`) are defined in `pyproject.toml` and `requirements.txt`. We recommend [Poetry](https://python-poetry.org/docs/) to install those dependencies and manage your virtual environment ([Poetry installation](https://python-poetry.org/docs/#installation)) or [pip](https://pip.pypa.io/en/latest/development/), but feel free to use a different way of installing dependencies and managing virtual environments if you have other preferences.
+## Set up the project
 
-#### Poetry
+### Clone the project
 
-```shell
-poetry install
-poetry shell
-```
-
-Poetry will install all your dependencies in a newly created virtual environment. To verify that everything works correctly you can run the following command:
+Start by cloning the example project:
 
 ```shell
-poetry run python3 -c "import flwr"
+git clone --depth=1 https://github.com/adap/flower.git _tmp \
+        && mv _tmp/examples/advanced-pytorch . \
+        && rm -rf _tmp \
+        && cd advanced-pytorch
 ```
 
-If you don't see any errors you're good to go!
-
-#### pip
-
-Write the command below in your terminal to install the dependencies according to the configuration file requirements.txt.
-
-```shell
-pip install -r requirements.txt
-```
-
-## Run Federated Learning with PyTorch and Flower
-
-The included `run.sh` will start the Flower server (using `server.py`),
-sleep for 2 seconds to ensure that the server is up, and then start 10 Flower clients (using `client.py`) with only a small subset of the data (in order to run on any machine),
-but this can be changed by removing the `--toy` argument in the script. You can simply start everything in a terminal as follows:
-
-```shell
-# After activating your environment
-./run.sh
-```
+This will create a new directory called `advanced-pytorch` with two sub-directories, each cotaining the same Flower app but one makes use of high-level components such as the `strategy` and the other uses the low-level API.
```

Comment:
```suggestion
This will create a new directory called `advanced-pytorch` with two sub-directories, each containing the same Flower app, but one makes use of high-level components such as the `Strategy` and the other uses the low-level API.
```